docs/en_US/README.md

```diff
@@ -0,0 +1,206 @@
+# MTProto
```

Comment:
Better to make symlink on root README (same to LICENSE file), but we do it later.